src/Controller/ResetPasswordController.php

```diff
@@ -90,33 +89,27 @@ public function reset(Request $request, UserPasswordEncoderInterface $passwordEn
         $user = $this->resetPasswordHelper->validateTokenAndFetchUser($token);
 
         //Reset password after token verified
-        $form = $this->createForm(PasswordResettingFormType::class);
+        $form = $this->createForm(ResetPasswordResetFormType::class);
```

Comment:
What about just calling this `ChangePasswordFormType`. It's actually a "generic" form type which could (and *should*) be re-used for a normal "change password" form.

Reply:
changed in maker 655c890a 

- [ ] verify templates and test

Reply:
Without adding to much complexity, perhaps asking if a change password form already exists and use that. Think something like that would be best to implement in a later version. There would be issues to overcome with naming conventions, etc...

NoteToSelf: adding a guesser similar to the interactive security helpers

Reply:
done